wp-content/themes/thinkery/style.css

```
@@ -221,6 +221,10 @@ blockquote {
 	margin: 30px;
 }
 
+.program-buttons {
+	test-align: center;
```

```suggestion
	text-align: center;
```

Is this necessary? Can't we center through the block settings / general WP justification classes?